src/components/course/CoursePage.jsx

```diff
@@ -198,6 +198,7 @@ const CoursePage = () => {
     courseEntitlements: courseState?.course?.entitlements,
     activeCourseRun: courseState?.activeCourseRun,
     userSubsidyApplicableToCourse,
+    canEnrollListPrice: subsidyAccessPolicyRedeemabilityData?.coursePrice,
```

Comment:
[curious] Should we also be pulling from `subsidyAccessPolicyRedeemabilityData?.coursePrice` for `courseListPrice` on line 191, preferring `subsidyAccessPolicyRedeemabilityData?.coursePrice` but falling back to `courseState?.activeCourseRun?.firstEnrollablePaidSeatPrice || getEntitlementPrice(courseState?.course?.entitlements))`?

Reply:
Good observation!